### 문제 1

- 2015년에 미국 플로리다주 고속도로 안전국과 자동차국에 의해 수집된 사고 기록이다. 표본오즈비, 상대위험도와 비율의 차이를 계산하고 해석하시오. 그리고 오즈비와 상대위험도가 근사적으로 같은 이유를 설명하시오. R이나 다른 통계패키지를 이용하여 분석하시오.

||치명적 사고<br>예|치명적 사고<br>아니오|
|:-:|:-:|:-:|
|안전장치<br>사용안함|433|8,049|
|안전벨트|570|554,883|

In [1]:
# install.packages("epitools")
library(epitools)

In [2]:
# 데이터 입력
data_matrix = matrix(c(433, 8049, 570, 554883), nrow=2, byrow=TRUE)
colnames(data_matrix) = c("치명적 사고: 예", "치명적 사고: 아니오")
rownames(data_matrix) = c("안전장치 사용안함", "안전벨트 사용")

data_matrix

치명적 사고: 예 치명적 사고: 아니오
안전장치 사용안함 433               8049             
안전벨트 사용     570             554883

In [3]:
# 이차원 분할표에서 π1, π2를 계산
prop.result = prop.table(data_matrix, margin=1)
prop.result

pi_1 = prop.result[1,1]
pi_2 = prop.result[2,1]

cat("π1:", pi_1, "\n")
cat("π2:", pi_2, "\n")

치명적 사고: 예 치명적 사고: 아니오
안전장치 사용안함 0.051049281     0.9489507          
안전벨트 사용     0.001026189     0.9989738

π1: 0.05104928 
π2: 0.001026189 


In [4]:
# 상대위험도 계산
risk_ratio = pi_1/pi_2
risk_ratio

[1] 49.74645

In [5]:
# 오즈비 계산
odd1 = pi_1 / (1-pi_1)
odd2 = pi_2 / (1-pi_2)
odd_ratio = odd1/odd2
odd_ratio

[1] 52.36879

- 오즈비는 52.369, 상대위험도는 49.746 으로 근사한 값을 가진다.

- 오즈비가 52.369라는 것은 안전벨트를 사용하지 않은 그룹에서 치명적 사고가 발생할 오즈가 안전벨트를 사용한 그룹에 비해 52.369배 크다는 것을 의미한다.

- 상대위험도가 49.746이라는 것은 안전벨트를 사용하지 않은 그룹에서 치명적 사고가 발생할 확률이 안전벨트를 사용한 그룹에 비해 49.746배 크다는 것을 의미한다.

- 오즈비와 상대위험도가 근사한 값을 가지는 이유는 안전벨트 사용여부와 관계없이 치명적 사고가 발생할 확률이 0에 가까운 값이기 때문이다.

### 문제 2
참고문헌에서 제시한 "오즈비와 상대위험도 관련 자료" 중 하나의 논문을 읽고 그 내용과 자기 생각을 A4 용지 1매로 작성하시오.

별도 첨부 파일로 제출

### 문제3
두 처리 (A, B)와 반응변수 (사망, 생존) 간의 오즈비가 2.0이라고 하자.

(1) "처리 A의 사망 확률은 처리 B에 비하여 2배이다"라는 해석이 왜 틀렸는지 설명하고 옳은 해석을 하시오.

- "처리 A의 사망 확률은 처리 B에 비하여 2배이다"이라는 해석이 틀린 이유는 그 해석이 상대위험도에 대한 해석이며, 오즈비에 대한 해석이 아니기 때문이다.
- 오즈비가 2.0일 때의 옳은 해석은 "처리 A의 오즈값이 처리 B의 오즈값보다 2배 크다"이다.


(2) (a)에서의 해석이 근사적으로 맞는 경우는 어느 경우인가?

- 처리 A, B의 사망 확률이 0에 가까운 값인 경우에는 오즈비와 상대위험도의 값이 근사한다. 이 경우에는 (a)에서의 해석이 근사적으로 맞다고 할 수 있다.

(3) 처리 A의 사망 오즈가 0.5일 때 처리 A와 처리 B의 사망 확률을 구하시오.

- 처리 A의 사망 오즈가 0.5 이고, 오즈비가 2가 되려면 처리 B의 사망 오즈는 0.25이다.
- 처리 B의 사망 오즈가 0.25일 때, 처리 B시 사망확률은 0.2이다.

<img src="https://latex.codecogs.com/svg.image?\frac{\pi_2}{1-\pi_2}=0.25"> <br>
<img src="https://latex.codecogs.com/svg.image?\Rightarrow\pi_2=0.2">

- 참고로, 처리 A의 사망 오즈가 0.5라 하면, 처리 A시 사망확률은 1/3 이다.